.azure-pipelines.yml

```diff
@@ -55,6 +55,7 @@ jobs:
   condition: eq(variables['Build.Reason'], 'PullRequest')
   pool:
     vmImage: 'macOS-11'
+  timeoutInMinutes: 180
```

Comment:
Is this accidentally included in this PR? It's not mentioned in the description.

Reply:
Ah yes, this was an action taken when jgraham's PR was not yet merged, so that the CI could run in its entirety to check for errors. I've removed it now.